package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "globalize-webpack-plugin",
-  "version": "3.0.0",
+  "version": "3.0.1",
```

Comment:
Above, the version is updated. Can we skip this from this PR please? It will be updated in the release. Thanks

Reply:
Sorry about that, fixing...